syllabus.md

```diff
@@ -11,20 +11,39 @@ The course begins with an introduction to probability theory, with pencil-and-pa
+- **Live Session** - Live sessions take place over Zoom, and can be accessed from [[ISVC]](https://learn.datascience.berkeley.edu/). During live session, students engage in activities that build upon the asynchronous videos and readings. These include discussions, collaborative problem solving, and programming exercises. We expect that you dial in from a quiet location, that you will arrive on time, and that you connect over both audio and video.  Most importantly, we ask that you treat all classmates with respect and help us create a supportive learning environment.
 
-- **After Live Session** - After live session, students complete the homework for the corresponding unit, as well as any tests or lab assignments. 
+- **Homework** - Most weeks of the course include a homework assignment. The homework for each week will be available Tuesday at 2pm on [[Github]](https://github.com/mids-w203). It will be due the following Tuesday at 2pm, and submitted via [[Gradescope]](www.gradescope.com). **It is important that you start the homework early, including before your live session, to give yourself a full week to complete it.** 
 
+- **Solution Files** - Solutions to each homework and each live session will be available on [[Github]](https://github.com/mids-w203). You learning is not complete when you solve a problem; it is important that you view the solution to solidify your knowledge and identify ways to improve your arguments. Homework solutions are located in the semester's "central" repository, and will be updated Thursdays at 2pm. Each live session repository will be located in a separate repository, with a name like unit_XX_ls_sol. This allows us to give each student access to the solution immediately after live session. 
+
+- **Tests** - The course includes two tests, located on [[Gradescope]](www.gradescope.com). The tests are made available during week 4 and week 6 of the course, and must be completed by Tuesday 2pm of the following week. There is a 3-hour time limit for each test. It is up to you to decide when to begin each test. Once you start the test on [[Gradescope]](www.gradescope.com), a 3-hour timer will begin to count down. Practice tests will be available to give you a sense of what the tests are like.
+
+- **Labs** - There are two labs that are multi-week, group-projects. Labs are available on [[Github]](https://github.com/mids-w203) and will be submitted via [[Gradescope]](www.gradescope.com). Groups are typically between three and four students. Each lab includes a complete analysis, from data wrangling and statistical procedures, to writing a persuasive report.
+
+- **Slack Channel** - We use a slack channel, datasci-203-2022-spring, for official course announcements and general discussion. This is a public channel within the ucbischool.slack.com workspace.
```

Comment:
We also will have section-specific slack channels that the live session instructor will announce. Be sure to join the section-specific slack channel.

Reply:
```suggestion
- **Slack Channel** - We use a slack channel, datasci-203-2022-spring, for official course announcements and general discussion. This is a public channel within the ucbischool.slack.com workspace. Please also join any section-specific slack channels that your live session instructor will announce.
```